test/e2e-kind.sh

```diff
@@ -13,6 +13,7 @@ readonly CLUSTER_NAME=chart-testing
 CT_VERSION=$1
 HELM_VERSION=$2
 GIT_REMOTE_NAME=${GIT_REMOTE_NAME:=origin}
+CURL_RETRY_OPTS="--max-time 30 --retry 20 --retry-connrefused"
```

Comment:
If there is no reason behind the numbers used here, I believe the values are too high. I suggest using `--retry-max-time` to set the total time before the entire request with retries is considered failed (maybe set to `60` so that we know a request can take 1 minute max?) with lower `--max-time` and `--retry` values.

Reply:
thanks for the tips! still playing around with the numbers, I'll lower `--max-time` and `--retry` and set `--retry-max-time`